app-office/drawio-desktop/drawio-desktop-22.1.2.ebuild

```diff
@@ -0,0 +1,137 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_PN="draw.io"
+MY_NODE_N="node-modules"
+MY_NODE_D="node_modules"
+DRAWIO_V="22.1.4"
+
+inherit desktop xdg
+
+DESCRIPTION="draw.io diagramming and whiteboarding desktop app"
+HOMEPAGE="https://www.drawio.com/"
+
+SRC_URI="
+	https://github.com/jgraph/drawio-desktop/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz
+	https://github.com/jgraph/drawio/archive/refs/tags/v${DRAWIO_V}.tar.gz
+	https://github.com/MocioF/distfiles/raw/main/${PN}-${MY_NODE_N}-${PV}-${DRAWIO_V}.tar.xz
+"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	>=net-libs/nodejs-16[ssl]
+	>=dev-libs/nss-3
+	app-accessibility/at-spi2-core
+	net-print/cups
+	x11-libs/libdrm
+	x11-libs/gtk+:3
+	x11-libs/libxkbcommon
+	media-libs/alsa-lib
+"
+
+MY_NODE_DIR="${S}/${MY_NODE_D}/"
+
+src_prepare() {
+	default
+	# We will use pre-generated npm stuff.
+	mv "${WORKDIR}/${MY_NODE_D}" "${MY_NODE_DIR}" || die "couldn't move node_modules"
+
+	# Remove unneeded modules with Pre-stripped files
+	rm -r "${MY_NODE_DIR}/7zip-bin" || die "couldn't remove 7zip-bin module"
+	rm -r "${MY_NODE_DIR}/app-builder-bin" || die "couldn't remove app-builder-bin module"
+
+	# Move drawio sources.
+	rm -r "${WORKDIR}/${P}/drawio" || die "couldn't remove old drawio dir"
+	mv "${WORKDIR}/drawio-${DRAWIO_V}" "${WORKDIR}/${P}/drawio" || die "couldn't move drawio sources"
+}
+
+src_install() {
+	# Copy node_modules dir
+	insinto "/usr/share/${PN}/"
```

Comment:
https://devmanual.gentoo.org/general-concepts/filesystem/index.html

/opt is mean for packages that cant installed normally.  

>  The /opt top-level should only be used for applications that do not conform to the standard filesystem layout. This particularly includes prebuilt software packages that expect being installed in a single directory. 

> The /usr/share directory is for architecture independent application data which is not modified at runtime.